assets/js/logging.js

```diff
@@ -0,0 +1,21 @@
+var colors = require('colors');
```

Comment:
I'm wondering what "assets" relates to dist-wise: We ship a source package, kolibri. All files that are distributed with kolibri have to reside inside it as "package data": But what does the /assets/ folder do?